cpp/velox/tests/ColumnarToRowTest.cc

```diff
@@ -83,7 +83,7 @@ class ColumnarToRowTest : public ::testing::Test {
     return rb;
   }
 
-  std::shared_ptr<facebook::velox::memory::MemoryPool> veloxPool_ = GetDefaultWrappedVeloxMemoryPool();
+  std::shared_ptr<facebook::velox::memory::MemoryPool> veloxPool_ = GetDefaultLeafWrappedVeloxMemoryPool();
```

Comment:
Can you use addLeafChild to track if which test hash memory leak?

Reply:
Sorry, I do not understand here. because I think we already call `addLeafChild` in `GetDefaultLeafWrappedVeloxMemoryPool` [here](https://github.com/oap-project/gluten/pull/1378/files#diff-163aa1d4bcffd7b4dd3b270eab81f55748d7e44fe185cb9a8c736f8f20b2d3ccR452).